### ISSUE TITLE
[iPad] iOS 11.3.1 ; 6th Generation iPad

### ISSUE THREAD

**Issue by AbdallAhmed** [Ominousness/MitsuhaXI#31]
Trying to run this tweak on a 6th generation (2018) iPad and only getting a pink bar in music apps (Spotify and SoundCloud). 

Will iPad support be added? I got the tweak from the repo if that matters (maybe I need to install from source?)

**Comment by Jonathing** [comment]
Yes, I am having a few issues on my iPad Air 2 (11.3.1) as well. I’ll compile the tweak from source and see if I get any different results. Although I know Java better than Objective-C, I’ll look at the code as well.

**Comment by Jonathing** [comment]
Yeah boi it’s a dependency so Cydia automatically installed it.
Nevertheless, AudioSnapshotServer is a backend tweak, so there should be
little to no problems with that.

**Comment by Ominousness** [comment]
Well AudioSnapshotServer is what feeds MitsuhaXI the audio data. If it's not working the wave is flat even after a few resprings (and mediaserverd restarts).

**Comment by Jonathing** [comment]
The wave is not flat on the iPad. The problem there is the frontend, so
basically how it looks. This is especially true since iPads have he ability
to rotate basically almost any GUI screen by default.

**Comment by Jonathing** [comment]
One way of combating that is to make a method that reloads and gets the
screen size upon the event of the iPad rotating, given that the screen is
on, of course.
